src/components/Header/HeaderAuth.jsx

```diff
@@ -0,0 +1,92 @@
+import React, { useState } from "react"
+import { NavLink, useLocation } from 'react-router-dom'
+import useScreenWidth from "./ListenerWidth"
+import LogoHead from "./LogoHead"
+import MenuAuthDeskTab from "./MenuAuthDeskTab"
+import {
+    ContainerHeader,
+    LeftBlock,
+    PointsBalance,
+    NumberBalance,
+    MenuHeaderBlock,
+    MobileHamburgerContainer,
+    MobileHamburgerLine,
+    MobileHamburgerbackground,
+    MobileHamburgerCrossBox,
+    MobileHamburgerCross,
+    MobileHamburgerCrossHolder,
+    MobileHamburgerSectionMenuUp,
+    MobileHamburgerSectionMenu
+} from "./Header.styled"
+import { MAIN_DARK, MAIN_BACKGROUND } from "../../assets/themes/colors"
```

Comment:
themes imports should be between libraries and components

Reply:
Oh sorry, I'll change the order of the imports)